.github/workflows/e2e.yaml

```diff
@@ -72,7 +64,7 @@ jobs:
       - if: steps.eval-params.outputs.run-type == 'github'
         run: |
           make dev/tools
-      - name: Free up disk space for the Runner
+      - name: "Github Actions: Free up disk space for the Runner"
```

Comment:
Why this?

Reply:
This job runs 2 options on github directly or through circleci. Other steps in this job are prefixed to make it easier to read

Reply:
OK I thought these names only show up in GH workflow runs, but :shrug: